### ISSUE TITLE
misoc.interconnect.stream -> DMAC | PRI

### ISSUE THREAD

**Issue by peteut** [peteut/migen-axi#6]
* stream sink
* ignored EOP
* request burst transfer via DMAC | PRI

Refer to ARM DDI 0424D.

**Comment by peteut** [comment]
The Linux driver expects a *single* request, not a *burst* request as currently implemented:

```
root@zedboard-zynq7:~# devmem2 0xf8003100
/dev/mem opened.
Memory mapped at address 0xb6f68000.
Read at address  0xF8003100 (0xb6f68100): 0x00000007
```

**Comment by peteut** [comment]
Fixes the type: a065766c634fd20a7f6ae2fbecd71cf66056e7e0.

**Comment by peteut** [comment]
Tested on Zedboard using w/ fb60d1dc3e32f381c10119cdfc861cf4255c3333.
